packages/xstate-react/src/useActor.ts

```diff
@@ -0,0 +1,22 @@
+import { ActorRef, Sender } from './ActorRef';
+import { EventObject } from 'xstate';
+import { useState, useEffect, useMemo } from 'react';
+
+export function useActor<TCurrent, TEvent extends EventObject>(
+  actorRef: ActorRef<TCurrent, TEvent>
+): [TCurrent, Sender<TEvent>] {
+  const [current, setCurrent] = useState(actorRef.current);
+
+  const send = useMemo(() => actorRef.send, [actorRef]);
```

Comment:
no need to memoize simple property access, it shouldn't be perf sensitive nor it should lead to a different reference being returned each time
```suggestion
  const send = actorRef.send;
```